spec/support/vcr.rb

```diff
@@ -14,6 +14,7 @@
   c.filter_sensitive_data('<EVSS_AWS_BASE_URL>') { Settings.evss.aws.url }
   c.filter_sensitive_data('<EVSS_BASE_URL>') { Settings.evss.url }
   c.filter_sensitive_data('<EVSS_DVP_BASE_URL>') { Settings.evss.dvp.url }
+  c.filter_sensitive_data('<FARADAY_VERSION>') { Faraday::Connection::USER_AGENT }
   c.filter_sensitive_data('<GIDS_URL>') { Settings.gids.url }
```

Comment:
My only comment here is potentially adding something to our documentation so others are aware of VCR sensitive data filters.